README.md

```diff
@@ -9,3 +9,4 @@ Various build tools used as part of the M3DB project
 
 1. [badtime](https://github.com/m3db/build-tools/blob/master/linters/badtime/README.md) - gometalinter plugin that detects inappropriate usage of the time.Time struct.
 2. [mockclean](https://github.com/m3db/build-tools/blob/master/utilities/mockclean/README.md) - CLI tool to make `mockgen` runs idempotent.
+2. [importorder](https://github.com/m3db/build-tools/blob/master/linters/importorder/README.md) - gometalinter plugin that detects accuracy of import ordering based on user specified patterns.
```

Comment:
this should be `3. [importorder] ...` 

also need to add this util to the `Makefile`, follow steps listed in https://github.com/m3db/build-tools/blob/master/DEVELOPMENT.md